src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_helpers.cc

```diff
@@ -0,0 +1,103 @@
+#include "rc_rpc_plugin/rc_helpers.h"
+#include "rc_rpc_plugin/rc_module_constants.h"
+#include "rc_rpc_plugin/rc_rpc_plugin.h"
+#include "application_manager/smart_object_keys.h"
+#include "application_manager/commands/command_impl.h"
+#include "application_manager/message.h"
+#include "utils/make_shared.h"
+
+namespace rc_rpc_plugin {
+CREATE_LOGGERPTR_GLOBAL(logger_, "RemoteControlModule");
+
+const std::function<std::string(const std::string& module_type)>
+RCHelpers::GetModuleTypeToDataMapping() {
+  auto mapping_lambda = [](const std::string& module_type) -> std::string {
+    static std::map<std::string, std::string> mapping = {
+        {enums_value::kRadio, message_params::kRadioControlData},
+        {enums_value::kClimate, message_params::kClimateControlData},
+        {enums_value::kSeat, message_params::kSeatControlData}};
```

Comment:
please make sure to update this mapping when other RC modules get merged into develop

Reply:
@yang1070  will take care of it